WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/ItemListSelectorCommand.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import UIKit
+
+/// Command to populate the IssueRefund item quantity list selector
+///
+final class ItemListSelectorCommand: ListSelectorCommand {
```

Comment:
nit: the name sounds a bit generic to me while the navigation bar title and init are specific to refund quantity. how about some name like `RefundItemQuantityListSelectorCommand`?

Reply:
👍